examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,615 @@
+    "        )\n",
+    "        return RAGWrappedScorer()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And now we can run our evaluation!\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Experiment Using gpt-3.5-turbo is running at https://www.braintrust.dev/app/braintrustdata.com/p/Coda%20RAG%20with%20ragas/experiments/Using%20gpt-3.5-turbo\n",
+      "Coda RAG with ragas [experiment_name=Using gpt-3.5-turbo] (data): 20it [00:00, 53025.34it/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "956aa08b6805436b9724b571ec60e54f",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Coda RAG with ragas [experiment_name=Using gpt-3.5-turbo] (tasks):   0%|          | 0/20 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "=========================SUMMARY=========================\n",
+      "90.45% 'ContextRecall'     score\n",
+      "68.23% 'AnswerCorrectness' score\n",
+      "\n",
+      "2.22s duration\n",
+      "\n",
+      "See results for Using gpt-3.5-turbo at https://www.braintrust.dev/app/braintrustdata.com/p/Coda%20RAG%20with%20ragas/experiments/Using%20gpt-3.5-turbo\n"
+     ]
+    }
+   ],
+   "source": [
+    "from braintrust import Eval\n",
+    "\n",
+    "eval_result = await Eval(\n",
+    "    name=\"Coda RAG with ragas\",\n",
+    "    experiment_name=f\"Using {QA_ANSWER_MODEL}\",\n",
+    "    data=qa_pairs[:20],\n",
+    "    task=generate_answer_e2e,\n",
+    "    scores=[RAGScorerWrapper(AnswerCorrectness), RAGScorerWrapper(ContextRecall)],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One difficulty of ragas is that it can be complex to understand how metrics were generated for a given datapoint, but with Braintrust we see each each constituent metric, any submetrics, and go down to the actual LLM calls used to generate the metric components.\n",
+    "\n",
+    "For exampe, here, we can see that AnswerCorrectness is computed by taking both a factuality and answer similarity score, and can examine the subscore computations - in this case, `factuality_score` and `answer_similarity_score`\n",
+    "\n",
+    "![ ragas metric computation ](assets/ragas_metric_computation.png)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Evaluating changes\n",
+    "\n",
+    "Now that we have our end-to-end system, and know how to use braintrust to examine the results, let's make some small changes and evaluate if they helped\n",
```

Comment:
Braintrust capitalization should be consistent (`Braintrust`)
Need a period after `helped`